_register.php

```diff
@@ -6,10 +6,13 @@
 use Shopware\OpenTelemetry\Instrumentation\ConnectionInstrumentation;
 use Shopware\OpenTelemetry\Instrumentation\DALInstrumentation;
 use Shopware\OpenTelemetry\Instrumentation\HttpClientInstrumentation;
+use Shopware\OpenTelemetry\Instrumentation\OpenSearchInstrumentation;
+use Shopware\OpenTelemetry\Instrumentation\ShopwareInstrumentation;
 use Shopware\OpenTelemetry\Instrumentation\SymfonyInstrumentation;
 use OpenTelemetry\SDK\Sdk;
+use Shopware\OpenTelemetry\Instrumentation\TwigInstrumentation;
 
-if (class_exists(Sdk::class) && Sdk::isInstrumentationDisabled(SymfonyInstrumentation::NAME) === true) {
+if (class_exists(Sdk::class) && Sdk::isInstrumentationDisabled(ShopwareInstrumentation::NAME) === true) {
```

Comment:
This can enable disabled instrumentations. Need to communicate this change during release.